source/includes/harvest/_eeoc.md

```diff
@@ -0,0 +1,80 @@
+# EEOC
```

Comment:
A rendered version if it makes it easier to read,

https://gist.github.com/jaydeesimon/17b06c24cb6a0f6841e7ee219a36462d